benches/plane.rs

```diff
@@ -24,6 +24,14 @@ pub fn downsample_8bit(c: &mut Criterion) {
   });
 }
 
+pub fn downsample_odd(c: &mut Criterion) {
+  let input = init_plane_u8(1919, 1079);
+  c.bench_function("downsample_odd", move |b| {
+    b.iter(|| {
+      let output = input.downsampled(960, 540);
+    })
+  });
+}
```

Comment:
```suggestion
}

```
It's better to add new line as in other functions.